### PR TITLE
896 infinite limits

### PR DESCRIPTION
**Pull request by adriguir**

### CODE REVIEW

File: dynawo/sources/Models/CPP/ModelNetwork/DYNModelDanglingLine.cpp [hunk 1]
```diff
@@ -89,17 +89,22 @@ modelType_("DanglingLine")  {
 
 
   double factorPuToA = sqrt(3.) * vNom / (1000. * SNREF);
+  double limit = 0.;
```
Comment (rosiereflo) [inline dynawo/sources/Models/CPP/ModelNetwork/DYNModelDanglingLine.cpp:92]: let's declare it only when used (inside the if)

File: dynawo/sources/Models/CPP/ModelNetwork/DYNModelLine.cpp [hunk 2]
```diff
@@ -135,16 +135,21 @@ modelType_("Line") {
...
   // current limits side 1
+  double limit = 0.;
   vector<shared_ptr<CurrentLimitInterface> > cLimit1 = line->getCurrentLimitInterfaces1();
   if (cLimit1.size() > 0) {
     currentLimits1_.reset(new ModelCurrentLimits());
     currentLimits1_->setSide(ModelCurrentLimits::SIDE_1);
     // Due to IIDM convention
-    double limit = cLimit1[0]->getLimit() / factorPuToA_;
-    currentLimits1_->addLimit(limit, cLimit1[0]->getAcceptableDuration());
+    if (cLimit1[0]->getLimit() < maximumValueCurrentLimit) {
```
Comment (rosiereflo) [inline dynawo/sources/Models/CPP/ModelNetwork/DYNModelLine.cpp:144]: same as above

File: dynawo/sources/Models/CPP/ModelNetwork/DYNModelDanglingLine.cpp (hunk 1 above)
Comment (adriguir) [reply dynawo/sources/Models/CPP/ModelNetwork/DYNModelDanglingLine.cpp]: Corrected

File: dynawo/sources/Models/CPP/ModelNetwork/DYNModelLine.cpp (hunk 2 above)
Comment (adriguir) [reply dynawo/sources/Models/CPP/ModelNetwork/DYNModelLine.cpp:143]: Corrected

**Pull request merged**